### ISSUE TITLE
Repairing pouches with essence inside resets essence count.

### ISSUE THREAD

**Issue by Daggerdoc** [jpeter17/easy-empty#10]
If you repair your pouches while they have essence in them, the count that tracks how much essence is inside the pouches will reset to zero despite there still being essence inside the pouches. If you have multiple pouches, all of them reset.

**Comment by tdufala** [comment]
I don't think this plugin does anything with counting how much essence is in the pouch. Probably a different plugin, like Item Charges? edit: I know this is old and this plugin hasn't been updated for a while. I'm thinking of giving it a revival though, so I was just browsing the open issues 😅
